### PR TITLE
Ongoing forgot password functionality 

### PR DESCRIPTION
**Pull request by izelnakri**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -100,6 +100,35 @@ app.post('/users', (req, res) => {
...
+    if (user) {
+      // 2 - send an email with a unique link
+      res.redirect('/');
+    } else {
+      res.redirect('/forgot-password');
+    }
+  });
+});
+
+app.get('/change-password/:uuid', (req, res) => {
```
Comment (JessicaML) [inline index.js]: uuid = 'unique user id' ?

Comment (elakorkmaz) [reply index.js]: yes

File: routes/authentication.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  }).then((userInDB) => {
+    bcrypt.compare(req.body.password, userInDB.passwordDigest, (error, result) => {
+      if (result) {
+        req.session.user = userInDB;
+        res.redirect('/');
+      } else {
+        res.redirect('/login');
+      }
+    });
+  }).catch(() => {
```
Comment (Zahra120) [inline routes/authentication.js:41]: why you did not pass error in catch?

Comment (izelnakri) [reply routes/authentication.js:41]: You are right, we could. I didn't pass it on purpose because we are not using that error parameter.

**Pull request closed without merging**